front/src/modules/ui/components/editable-cell/types/EditableRelation.tsx

```diff
@@ -139,6 +149,72 @@ export function EditableRelation<
     setIsSomeInputInEditMode(false);
   }
 
+  const [selectedIndex, setSelectedIndex] = useState(0);
+  useHotkeys(
+    'down',
+    () => {
+      setSelectedIndex((prevSelectedIndex) =>
+        Math.min(
+          prevSelectedIndex + 1,
+          (filterSearchResults.results?.length ?? 0) - 1,
+        ),
+      );
+    },
+    {
+      enableOnContentEditable: true,
```

Comment:
I feel we should extract this into our own useHotKeys hooks. we could expose an api where we don't need to pass this config everytime and this would make it more readable. For a future PR